core/domain/image_services_test.py

```diff
@@ -120,9 +119,19 @@ def is_different(file1, file2):
         with self.swap(constants, 'DEV_MODE', False):
             compressed_image = (
                 image_services.compress_image(self.jpeg_raw_image, 0.5))
+
+        # I need to open and save it specifically using PIL since the "golden
```

Comment:
Not clear to the reader who "I" is (they'll be looking at the code, not at your PR). Can you write this more "objectively"?

Reply:
Done